code/modules/surgery/organs/subtypes/standard.dm

```diff
@@ -17,6 +17,19 @@
 	encased = "ribcage"
 	convertable_children = list(/obj/item/organ/external/groin)
 
+/obj/item/organ/external/chest/emp_act(severity)
+	..()
+	if(!is_robotic() || emp_proof || !tough) // Augmented chest suffocates the user on EMP.
```

Comment:
I think I would add a check for `owner` here so you don't have to use so many `?.` operators. Also, with the code as is, the `to_chat` doesn't check if owner is null, which can runtime.